src/words/Letter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {isNull} from 'lodash';
+
+function setColor(correct) {
+  if (isNull(correct)) {
+    return 'black'
```

Comment:
Just to confirm that I understood the purpose of this: the default color of the words are `black`, right?

Reply:
correct